cms/djangoapps/contentstore/management/commands/tests/test_reindex_courses.py

```diff
@@ -0,0 +1,130 @@
+""" Tests for course reindex command """
+import ddt
+from django.core.management import call_command, CommandError
+import mock
+
+from xmodule.modulestore import ModuleStoreEnum
+from xmodule.modulestore.django import modulestore
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from common.test.utils import nostderr
+from xmodule.modulestore.tests.factories import CourseFactory, LibraryFactory
+
+from opaque_keys import InvalidKeyError
+
+from contentstore.management.commands.reindex_course import Command as ReindexCommand
+from contentstore.courseware_index import SearchIndexingError
+
+
+@ddt.ddt
+class TestReindexCourse(ModuleStoreTestCase):
+    """ Tests for course reindex command """
+    def setUp(self):
+        """ Setup method - create courses """
+        super(TestReindexCourse, self).setUp()
+        self.store = modulestore()
+        self.first_lib = LibraryFactory.create(
+            org="test", library="lib1", display_name="run1", default_store=ModuleStoreEnum.Type.split
+        )
+        self.second_lib = LibraryFactory.create(
+            org="test", library="lib2", display_name="run2", default_store=ModuleStoreEnum.Type.split
+        )
+
+        self.first_course = CourseFactory.create(
```

Comment:
Is there a reason we insist on a split store here? For Library it has to be split, but I don't think we need to hardcode this for courses.